plugins/3MFReader/WorkspaceDialog.qml

```diff
@@ -300,6 +300,20 @@ UM.Dialog
                     }
                 }
 
+                Row
+                {
+                    id: dropToBuildPlate
+                    width: parent.width
+                    height: childrenRect.height
+                    spacing: UM.Theme.getSize("default_margin").width
+                    UM.CheckBox
+                    {
+                        text: catalog.i18nc("@text:window", "Drop models to buildplate")
+                        checked: UM.Preferences.getValue("physics/automatic_drop_down") == True
+                        onCheckedChanged: UM.Preferences.setValue("physics/automatic_drop_down", checked)
```

Comment:
While I think the general approach of this ticket is good, the setting does more than just take effect when loading a model into Cura: It also handles what happens if you make a model float regardless of opening.

Also; what happens if you open a model, change the checkbox to be different from your preference, then load a (floating) STL? (Or 'non-workspace' 3MF, or other...)

So here should probably be a dropdown operation attached for the just opened models somehow (in case of a workspace, that'll remove all others from the buildplate, so it can just be all models loaded), instead of changing the entire preference each time we load a workspace.

Reply:
32d9e6b45c56f65beea413248849b8599e62c30a
I made a separate preference variable that silently mimics drop_to_buildplate value until a user selects otherwise for that model. Solves this issue 💯